cli/bundle/bundle_test.go

```diff
@@ -1 +1,33 @@
 package bundle
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+func TestBundle(t *testing.T) {
```

Comment:
If we aim for test coverage, we probably just test individual functions in those packages. The test engine is not smart enough to capture code path covered by exec.Command in a separate process :(